File: game.py
print('Welcome Tresure Island ')
print('Your mission is to find the tresure !')

print('''
 .-._                                                   _,-,
  `._`-._                                           _,-'_,'
     `._ `-._                                   _,-' _,'
        `._  `-._        __.-----.__        _,-'  _,'
           `._   `#==="""           """===#'   _,'
              `._/)  ._               _.  (\_,'
               )*'     **.__     __.**     '*( 
               #  .==..__  ""   ""  __..==,  # 
Deelkar        #   `"._(_).       .(_)_."'   #
''')
direction = input('You/e in the middle of the way "left" or "right"').lower()

if direction == "left":
   left =  input('You have came to the lake , "swim" or "wait" \n').lower()
   if left == "wait":
       wait = input('You are in the final you have 3 rooms Choose : "yellow" "green"  "red" \n').lower()
       if wait == "yellow":
           print('You are Winner')
       elif wait == "red":
          print('You are buried , Game over')
       elif wait == "green":
        print('Game over , you choked on the green liquid ')
   else:
       print('Game over')
else:
    print('You fallaen in a hole , Game over !')

File: game.js
console.log('Welcome to the game ')
console.log('Treasure Game , ')

let direction = prompt('You are in the middle of tha Island , Choose "left" or "right"')

let getFunc = ()=>{
    if (direction === "left"){
        let left = prompt('You are near the lake boat is coming , "wait" or "swim"')
        if (left === "wait"){
            let rooms = prompt('You are arrived finally , Choose room "yellow" "green" "red"')
            if(rooms === "yellow"){
                console.log('You are Winner , Tresure')
            }
            else if (rooms === "red"){
            console.log('You are butied Game over ')
            }
            else 
            {
                console.log('Game over , you choked on the green liquid')
            }
        }
        else {
            console.log('You are drawned')
        }
    }
    else {
        console.log('You are fallen to the hole !')
    }
}
getFunc()